### PR TITLE
libs/react: add checkbox and expandable components (DEV-1080)

### PR DESCRIPTION
**Pull request by tglaz**
### Filter Shelter Results
https://betterangels.atlassian.net/browse/DEV-1080

Partial PR
* expandable container
* checkbox + checkboxGroup
* add `--color-primary-95` to shelter css

screnshot:

image

## Summary by Sourcery

Add checkbox and expandable container components to the React library.

New Features:
- Added `Checkbox` and `CheckboxGroup` components for form elements.
- Added `ExpandableContainer` component to display expandable content sections.

### CODE REVIEW

File: libs/assets/src/icons/svg/check.svg
Comment (mikefeldberg) [inline libs/assets/src/icons/svg/check.svg:1]: we have this icon in`libs/expo/shared/icons/src/assets/third_party/dariush_habibpour/svg/iconamoon:check-bold.svg`

File: libs/assets/src/icons/svg/chevronDown.svg
Comment (mikefeldberg) [inline libs/assets/src/icons/svg/chevronDown.svg:1]: pretty sure we're using this under a CCBY 4 license, so we have to add it to our attribution list for this creator. And we probably need to colocate it with the others, in `libs/expo/shared/icons/src/assets/third_party/dave_gandy/svg/`

Comment (tglaz) [reply libs/assets/src/icons/svg/chevronDown.svg:1]: I created https://betterangels.atlassian.net/browse/DEV-1163 for this.

Comment (tglaz) [reply libs/assets/src/icons/svg/chevronDown.svg:1]: I need to get the info from design. I pulled the svg from Figma cause we don't have that one in our files

Comment (tglaz) [reply libs/assets/src/icons/svg/chevronDown.svg:1]: and if we need to update all licenses now, i will remove them and can handle that later.

**Comment by tglaz** [comment]
closing, will return with another PR

**Pull request closed without merging**